controller/main.py

```diff
@@ -136,11 +135,36 @@ def message_received(client, server, message):
         )
 
 
-proxy.start()
-binaries.explore()
-server = WebsocketServer(PORT)
-server.set_fn_new_client(new_client)
-server.set_fn_client_left(client_left)
-server.set_fn_message_received(message_received)
-print("websocket server running ")
-server.run_forever()
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-v", "--verbosity", action="count", default=0)
+    parser.add_argument("--home")
```

Comment:
Is `home` is a good name? :thinking. How about `working_dir`?

Reply:
Ok, will change.

Reply:
changed to --work-dir